### DISCUSSION TITLE
Difference results of Faster-Whisper wrt Whisper for 'no_speech_threshold'

### DISCUSSION THREAD

**Discussion by raulqf** [SYSTRAN/faster-whisper#316] [General]
I've been working with an audio clip in spanish that contains speech along large silence segments. Faster-whisper transcription is failing, if the 'no_speech_threshold' is set to 0.6, its default value.  I've tested, large-v2 and medium obtaining a similar behavior. However, if you increment it to 1.0, the transcription is correct.

I've compared the  results with Whisper and the transcription is correct using the default values. 

If anyone wants to play with it, you could download the audio clip from here:

https://www.dropbox.com/s/1r4cjvdkqcc0nr1/test_audio.wav?dl=0

**Comment by guillaumekln** [comment]
I think you are using the master version of openai/whisper which changed the behavior of `no_speech_threshold`: https://github.com/openai/whisper/commit/e334ff141d5444fbf6904edaaf408e5b0b416fe8

We prepared the same change in #225 but I wanted to wait for OpenAI to cut a new version before merging it. If you use this branch you should get the same transcription as openai-whisper.

However, note that faster-whisper has a more robust way to deal with silence with the argument `vad_filter=True`.
